packages/loader/driver-utils/src/readAndParse.ts

```diff
@@ -13,9 +13,9 @@ import { IDocumentStorageService } from "@fluidframework/driver-definitions";
  * @param id - the id of the blob to read and parse
  * @returns the object that we decoded and JSON.parse
  */
-export async function readAndParse<T>(storage: Pick<IDocumentStorageService, "read">, id: string): Promise<T> {
-    const encoded = await storage.read(id);
-    const decoded = fromBase64ToUtf8(encoded);
+export async function readAndParse<T>(storage: Pick<IDocumentStorageService, "readBlob">, id: string): Promise<T> {
+    const blob = await storage.readBlob(id);
+    const decoded = bufferToString(blob, "utf8");
     return JSON.parse(decoded) as T;
 }
 
```

Comment:
I'm not sure about readAndParseFromBlobs, can `blobs` be an array of buffers? that way we can get rid of fromBase64ToUtf8 here and use bufferToString instead

Reply:
That is to read blobs from a snapshot taken in detached container where we store the contents as base64. 
Here is the code: https://github.com/microsoft/FluidFramework/blob/6cfc86f109bed61f24560acb644e7c599713a269/packages/loader/container-loader/src/utils.ts#L57

So right now it is correct. If we want to change then we have to change at both places. For this PR let it be this way.